draft-zip-hybrid-deferred-devfund.md

```diff
@@ -0,0 +1,413 @@
+independent teams dedicated to Zcash. Major funding decisions should be based,
+to the extent feasible, on inputs from domain experts and pertinent
+stakeholders.
+
+**Value of ZEC**: In case the value of ZEC jumps, ZCG should not wastefully use
+excessive amounts of funds. Conversely, given market volatility and eventual
+halvings, it is desirable to create rainy-day reserves.
+
+**Funding Stream**: The ZCG funding stream should conform to the standards of
+ZIP 207 and ZIP 214.
+
+## Specification
+
+Consensus changes implied by this specification are applicable to the Zcash
+Mainnet. Similar (but not necessarily identical) consensus changes SHOULD be
+applied to the Zcash Testnet for testing purposes.
+
+### Hybrid Deferred Dev Fund Allocation
+
+The Zcash Development Fund SHALL be enacted in Network Upgrade 6 and last for 2
+years. The Dev Fund would consist of 20% of the block subsidies, split into 2
+slices:
+
+* 60% to a deferred Dev Fund “lockbox,” which will be allocated in the future
+  once consensus is reached on fund allocation or a new non-direct funding
+  model is established and implemented.
+* 40% to Zcash Community Grants to fund independent teams in the Zcash ecosystem.
+
+### Deferred Dev Fund Lockbox
+
+<u>Deferred Development Fund Chain Value Pool Balance</u>
+
+A new development fund value pool balance, denominated in zatoshis, will be
+created. This balance will be initialized to zero at the NU6 activation block.
+
+<u>Coinbase Transaction Adjustments</u>
+
+For coinbase transactions, a specified proportion of the block subsidy will be
+added to the deferred development fund chain value pool balance and subtracted
+from the transparent transaction value pool. This subtraction will occur after
+block reward value is logically added to the transparent value pool. The exact
+proportions (numerator/denominator) of the block subsidy allocated to the
+deferred development fund must be defined to ensure the correct allocation at
+every block.
+
+<u>Proportion Specification</u>
+
+The proportions of the block subsidy allocated to the deferred development fund
+and any existing funding streams must add up to the intended total at every
+block. Before NU6 activation, the actual percentage allocated to the deferred
+development fund will be zero. After NU6 activation, |percentage| of the block
+reward will be allocated to the deferred development fund.
+
+<u>Backwards Compatibility</u>
+
+Full validators can implement the reserve consensus rules without NU gating.
+Prior to NU6 activation, the percentage allocated to the reserve pool can be
+set to zero or be defined as an empty set, similar to existing funding streams.
+Additionally, another implementation option would be to define a new type of
+funding stream that directs the value to the in-protocol development fund
+instead of constructing a UTXO. This approach allows for flexibility in
+handling the allocation and ensures compatibility with existing mechanisms.
+
```

Comment:
This should be in a rationale section, if included. For now, sufficient to delete from the specification section.
```suggestion
```